contrib/compactionEta.sh

```diff
@@ -0,0 +1,49 @@
+#/bin/bash
+
+# Description: a script to see how much time is left on major 
+#              compactions on cassandra nodes
+#
+# Instructions:
+#
+# a. To run for a single machine, replace 'domain.com' with the appropriate domain.  
+#    You can change the port or leave it blank to use the default port for nodetool:
+#
+#       ssh db01.domain.com 'bash -s' < ./compactionEta.sh [port]
+#
+# b. To run for multiple machines, here is a one-liner example. This assumes you number 
+#    your hosts like my team does, but hopefully you get the idea.
+#
+#       for x in {00..31}; do ssh db$(printf "%2.2d" $x).domain.com 'bash -s' < ./compactionEta.sh [port]; done
+#
+# c. Or, to run on multiple machines a little faster, install GNU parallel 
+#    and run the command below. Again, replace 'domain.com' and also <start> 
+#    and <end> with the first and last servers you want to check.
+# 
+#       parallel -j20 --no-notice "ssh db{1}.domain.com 'bash -s' < ./compactionEta.sh [port]" ::: $(seq -w <start> <end>) | sort
+
```

Comment:
`set -euo pipefail`, for conservative behavior (fail on non-zero command exit, fail on undefined variable, etc)?